packages/react-day-picker/src/components/Table/Table.tsx

```diff
@@ -42,6 +44,7 @@ export function Table(props: TableProps): JSX.Element {
   const FooterComponent = components?.Footer ?? Footer;
   return (
     <table
+      id={props.id}
```

Comment:
I don't think we need a prop here. Since we have the `props.displayMonth`, we can use it to calculate the unique id string, which could be `id + year + month`:

```suggestion
      id={id ? `{id}-${props.displayMonth.fullYear}-${props.displayMonth.month}` : undefined|
```

Where `id` comes from the `useDayPicker` hook:

```diff
-    ISOWeek
+    ISOWeek,
+    id
  } = useDayPicker();
```

Reply:
@gpbl If I'm understanding this correctly, this means that the id will continue to change for the displayed displayed tables depending on the displayed months. This is probably not a good idea because it means that we have to manage the displayed month state outside of the datepicker to be able to use the ids anywhere else. The current implementation is good because the displayed months are irrelevant, we just need the ids of the displayed tables.

Reply:
Oh sure, it makes sense. Thanks for the heads up.